c2corg_api/scripts/migration/documents/reports.py

```diff
@@ -8,7 +8,7 @@
 class MigrateReports(MigrateDocuments):
```

Comment:
It would be more consistent to rename the file to ``xreports.py`` and the class to ``MigrateXreports``.